### ISSUE TITLE
Templates: Able to update all (or multiple) Terms of Use + Access 

### ISSUE THREAD

**Issue by eaquigley** [IQSS/dataverse#2585]
Investigate adding a way to update all (or multiple) datasets using the same template to change the Terms of Use + Access without having to go into each dataset and do it.

**Comment by jggautier** [comment]
I'm working with someone who would like to update the Terms of Use for multiple datasets in bulk. The SWORD API lets users replace Terms of Use metadata, but it doesn't let users replace only the values in that metadata field: "ALL metadata (title, author, etc.) will be replaced, including fields that can not be expressed with “dcterms” fields." The Native API lets users replace values in metadata fields selectively, but it doesn't allow users to change values in any of the Terms fields (Terms of Use, Confidentiality Declaration, etc.).

Could we make it possible to change Terms fields using the Native API? it seems like this issue suggests a way to make bulk edits through the UI, but maybe adding it to the APIs first would help.

**Comment by pdurbin** [comment]
@jggautier yes but a dedicated issue called something like "allow terms to be changed via native API" would be nice.

I do think adding additional capabilities to the API is the right approach rather than building a GUI first.

Also, as we discussed, a workaround for now, in the absence of being able to update terms via the native API might be:

- serialize your dataset as native dataverse JSON format.
- change the terms of use via SWORD
- reupload your metadata from the JSON backup you made

The reason for backing up and re-applying the native JSON format is that with SWORD your metadata will be overwritten with very few (Dublin Core) fields. At least, that's how I believe it works. It's been a while since I've tested this. 😄

**Comment by jggautier** [comment]
I finally had the chance to test this workaround, and I don't think it will work right now, as of Dataverse 4.18.1. The problem is with the third step, "reupload your metadata from the JSON backup you made." I used the [update metadata API endpoint](http://guides.dataverse.org/en/latest/api/native-api.html#update-metadata-for-a-dataset) on a dataset that had Terms of Use metadata, and the Terms of Use metadata was removed.

**Comment by pdurbin** [comment]
@jggautier weird. Thanks for opening this issue by the way:

- Allow terms metadata to be changed with Native API #5899 

I imagine as part of that issue we would probably implement CRUD (Create, Read, Update, Delete) for terms so we the terms of use metadata would need to be available on "Read".

**Comment by pdurbin** [comment]
> @jggautier weird. Thanks for opening this issue by the way:
> 
>     * Allow terms metadata to be changed with Native API

That issue was closed by the semantic API PR:

- #7414

Does that mean this issue can be closed too? That is, can we update terms for datasets via the semantic API and not worry about the templates?
